src/components/form/form.js

```diff
@@ -49,47 +49,59 @@ class ManualEntryForm extends Component {
         }
 
         else if (this.state.barcodeValue.length < 12 | this.state.barcodeValue.length > 13) {
-            console.log(`The input length is ${this.state.barcodeValue.length}`)
             error = "That is not a valid barcode"
         }
        
         this.setState({barcodeError: error}, () => {
             console.log(`The error is: ${this.state.barcodeError}`)
+            if (this.state.barcodeError == null) {

```

Comment:
This is the section where the server teams API is called. It is very similar to the section of code in Scanner.js, I intend to refactor the API calls into the "utils" folder eventually.